terraform/environments/data-platform/api.tf

```diff
@@ -186,4 +192,39 @@ resource "aws_api_gateway_integration" "get_glue_metadata" {
     "integration.request.querystring.database" = "method.request.querystring.database",
     "integration.request.querystring.table"    = "method.request.querystring.table"
   }
-}
\ No newline at end of file
+}
+
+
+# create data product metadata API endpoint
+
+resource "aws_api_gateway_resource" "create_data_product_metadata" {
+  parent_id   = aws_api_gateway_rest_api.data_platform.root_resource_id
+  path_part   = "register_data_product"
```

Comment:
There was a discussion the other day about adopting RESTful paths - are we ready to do that yet, or is this blocked on Alex's ticket? Just thinking this is likely to be shortened to `data_product` or `data_products`, and register is implied from the POST method

Reply:
yeah, probably want to wait until we have a fuller picture